src/containers/org/components/RevokeRolesFromMembersBody.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * @flow
+ */
+
+import React, { useState } from 'react';
+
+import { Map } from 'immutable';
+import { useRequestState, useStepState } from 'lattice-utils';
+import { useDispatch } from 'react-redux';
+import type { Role, UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import BulkRoleSelectionStep from './BulkRoleSelectionStep';
+import ConfirmStepBody from './ConfirmStepBody';
+
+import { ORGANIZATIONS } from '../../../core/redux/constants';
+import { getUserProfile } from '../../../utils';
+import { REVOKE_ROLES_FROM_MEMBERS, revokeRolesFromMembers } from '../actions';
+
+const RESET_ACTIONS = [REVOKE_ROLES_FROM_MEMBERS];
+
+type Props = {
+  members :Map;
+  onClose :() => void;
+  organizationId :UUID;
+  roles :Role[];
+};
+
+const RevokeRolesFromMembersModalBody = ({
+  members,
+  onClose,
+  organizationId,
+  roles,
+} :Props) => {
+  const dispatch = useDispatch();
+  const [step, stepBack, stepNext] = useStepState(2);
+  const requestState :?RequestState = useRequestState([ORGANIZATIONS, REVOKE_ROLES_FROM_MEMBERS]);
+  const [selectedRoles, setSelectedRoles] = useState(Map());
+
+  const handleSelectRole = (role :Role) => {
+    const { id } = role;
+    const newSelection = selectedRoles.has(id)
+      ? selectedRoles.remove(id)
+      : selectedRoles.set(id, role);
```

Comment:
I recommend using `Set<UUID>` instead of `Map<UUID, Role>` because you're not actually using the role object for anything.